### PR TITLE
Remove /var/log/*/*.log after all components are stopped

### PR DESCRIPTION
**Pull request by guillaumerose**
Fixes https://github.com/code-ready/snc/issues/295

It will remove around 300MB of logs uncompressed.

### CODE REVIEW

File: createdisk.sh [hunk 1]
```diff
@@ -81,6 +81,9 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl rmp $(sudo crictl
 # Remove pull secret from the VM
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -f /var/lib/kubelet/config.json'
 
+# Remove audit logs
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -f /var/log/kube-apiserver/*'
```
Comment (praveenkumar) [inline createdisk.sh:85]: @guillaumerose what if we just delete everything from `/var/log` does those directory needs to be present for some container mount?

Comment (guillaumerose) [reply createdisk.sh:85]: good point, will try!

**Comment by guillaumerose** [comment]
It works fine if I remove everything. PTAL.

File: createdisk.sh [hunk 2]
```diff
@@ -81,6 +81,9 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl rmp $(sudo crictl
 # Remove pull secret from the VM
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -f /var/lib/kubelet/config.json'
 
+# Remove audit logs
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo find /var/log/ -iname "*.log" -exec rm -f {} \;'
```
Comment (praveenkumar) [inline createdisk.sh:85]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

File: createdisk.sh (hunk 2 above)
Comment (cfergeau) [reply createdisk.sh:85]: Looks like the comment/commit log were not updated when this PR changed from removing audit logs to removing /var/log?

**Pull request merged**